src/static/client/app/signup/signup.component.scss

```diff
@@ -1,11 +1,15 @@
+@import '../common.scss';
+
 @import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;600;700&display=swap');
 @import '../common.scss';
+@import '../common.scss';
 
-.layout {
-  display: flex;
-  justify-content: center;
-  align-items: center;
-  height: 100vh;
+body, html {
+  font-family: 'Poppins', sans-serif; 
+}
+
+.signup {
+  @extend .display-flex-center;
```

Comment:
can you do this for the other signup class i left a comment on? This is good!

Or just use the display-flex-center class if you aren't adding any new properties.